packages/docs/src/README.md

```diff
@@ -26,7 +30,17 @@ const app = Vue.createApp(App)
 app.use(Vuelidate)
 ```
 
-Then in your component you can create a `validations` method and define your validation rules.
+Now that Vuelidate is registered as a global plugin, create a `validations` method inside your component and define your validation rules.
+
+First, import the validators that you want to use from `@vuelidate/validators`.
+
+```js
+import { required } from '@vuelidate/validators'
+```
+
+Next, add the `validations` method, it should return an object that matches the structure of your state. In this example, we have a field `name`, which is declared inside of the component's `data()`.
```

Comment:
I don’t think we should call `validations` a method. It can be a function, but can also be just an object (when you don’t need `this` like in this example. Also, using the name `method` suggest adding a `validations` method to `methods`.